### PR TITLE
skipper: update hostname-credentials-controller

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
The logic previously implemented by secret-combiner was moved into hostname-credentials-controller.

### CODE REVIEW

File: cluster/manifests/deletions.yaml [hunk 1]
```diff
@@ -4,6 +4,20 @@ pre_apply:
   namespace: kube-system
   kind: Deployment
 
+# TODO: remove after rollout
```
Comment (szuecs) [inline cluster/manifests/deletions.yaml:7]: link to PR would be good

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**